spec/unit/platform/query_helpers_spec.rb

```diff
@@ -18,46 +18,52 @@
 
 require "spec_helper"
 
-describe "Chef::Platform#supports_dsc_invoke_resource?" do
-  it "returns false if powershell is not present" do
-    node = Chef::Node.new
-    expect(Chef::Platform.supports_dsc_invoke_resource?(node)).to be_falsey
-  end
-
-  ["1.0", "2.0", "3.0", "4.0", "5.0.10017.9"].each do |version|
-    it "returns false for PowerShell #{version}" do
+if ChefUtils.windows?
+  describe "Chef::Platform#supports_dsc_invoke_resource?" do
```

Comment:
this should be `describe "Chef::Platform#supports_dsc_invoke_resource?", :windows_only do`

Reply:
The syntax is defined in the `spec/spec_helper.rb` file and its useful to just grep through the source to see how its used.